.github/workflows/lock-threads.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Lock Closed Threads'
+
+on:
+  schedule:
+    - cron: '0 7 * * *'  # 7am UTC, daily
+  workflow_dispatch:
+
+permissions:
+  issues: write
+  pull-requests: write
+
+concurrency:
+  group: lock
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
+        with:
+          issue-lock-inactive-days: '30'
```

Comment:
The `inactive-days` can be numeric values:

```yaml
    issue-lock-inactive-days: 30
    pr-lock-inactive-days: 15
```

Reply:
Yea, and this is fine too! It's not like we'll be touching this file too often.